<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChat</class>
 <widget class="QWidget" name="GroupChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="groupMsg_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userMsg_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsgButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QListWidget" name="groupMember_listWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>111</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
